packages/frontend/apps/ios/docs/onboarding-paywall-plan.md

```
@@ -0,0 +1,204 @@
+   - 标识为 `false`（默认 / 首启）→ 进入 **onboarding 流程**（可选衔接 paywall）
+   - 标识为 `true` → 直接进入 `.main`（当前的 AFFiNE Web 主界面）
+2. onboarding 结束后写入标识为 `true`，后续启动不再展示。
+3. 评估并接入 paywall（复用现有 `AffinePaywall` 包）。
+
+## 2. 现状分析（事实，非假设）
+
+### 2.1 启动链路
+
+- `App/App/Base.lproj/Main.storyboard` 指定入口控制器 `RootViewController`。
+- `RootViewController`（`UINavigationController` 子类）在 `commitInit()` 中硬编码：
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)